### PR TITLE
chore: CODEOWNERS audit

### PR DESCRIPTION
**Pull request by rsamborski**
## Description

Fixes [b/275440264](http://b/275440264)

Updates after a cross-repo audit in preparations for the upcoming FixIt.

## Checklist
- [X] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -56,24 +55,25 @@
 /iam/                                  @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-infra 
 /kms/                                  @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-infra
 /securitycenter/                       @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-infra 
+/mediacdn/                             @justin-mp @GoogleCloudPlatform/dee-infra @GoogleCloudPlatform/go-samples-reviewers
```
Comment (telpirion) [inline .github/CODEOWNERS:58]: Please do not add individuals to CODEOWNERS.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -56,24 +55,25 @@
...
 /container/                            @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-platform-ops
-/logging/                              @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-observability 
-/monitoring/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-observability
-/trace/                                @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-observability 
+/logging/                              @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-platform-ops 
+/monitoring/                           @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-platform-ops
+/trace/                                @GoogleCloudPlatform/go-samples-reviewers @GoogleCloudPlatform/dee-platform-ops 
 
 # Non-DEE owner
+/auth/                                 @GoogleCloudPlatform/googleapis-auth @GoogleCloudPlatform/go-samples-reviewers
```
Comment (telpirion) [inline .github/CODEOWNERS:67]: The @GoogleCLoudPlatform/googleapis-auth team isn't a part of this repo.

File: .github/CODEOWNERS (hunk 1 above)
Comment (telpirion) [reply .github/CODEOWNERS:58]: nit: please keep "go-samples-reviewers" as first in the list for tidiness reasons.

Comment (rsamborski) [reply .github/CODEOWNERS]: I support the rule of not adding individuals to CODEOWNERS, but I wanted to make an exception in this case. We are assisting the product team with these samples therefore dee-infra is here plus TL from product. Creating a teamsynced group with just @justin-mp as a member seems to be an overkill.

I have switched the order as suggested.

File: .github/CODEOWNERS (hunk 2 above)
Comment (rsamborski) [reply .github/CODEOWNERS:67]: Unfortunately I am not an admin of this repo therefore please add write access to necessary groups per [this comment](https://github.com/GoogleCloudPlatform/golang-samples/pull/2988#issuecomment-1517593799).

**Comment by rsamborski** [comment]
Closing this PR and moving to a clean version #2992 due to issues with conflict resolution.

**Pull request closed without merging**